fastembed/image/transform/operators.py

```diff
@@ -157,6 +182,23 @@ def _get_resize(transforms: list[Transform], config: dict[str, Any]):
                         resample=config.get("resample", Image.Resampling.BICUBIC),
                     )
                 )
+        elif mode == "JinaCLIPImageProcessor":
+            resample = (
+                Compose._interpolation_resolver(config.get("interpolation"))
+                if isinstance(config.get("interpolation"), str)
+                else config.get("interpolation", Image.Resampling.BICUBIC)
+            )
```

Comment:
it is hard to read, why do we do `config.get('interpolation')` 3 times? We cna just write it to a variable once and reuse